e2e/visual-regression/all.e2e.ts

```diff
@@ -17,7 +18,14 @@ describe('components', () => {
 
       cy.visit(`/iframe.html?id=${id}`);
 
-      cy.get('body').should('be.visible').matchImageSnapshot(name);
+      // use slightly higher timeout because sometimes the iframe is slow
+      cy.get('body', { timeout: 10000 }).should('have.class', theme);
+
+      // give React one cycle to run effects and timeouts and stuff,
+      // and give the browser one requestFrame
+      cy.wait(20);
```

Comment:
Wondering whether this test could be more deterministic? Perhaps display a success state when the effects have finished running or the iFrame has loaded.

Reply:
The reason we need this is for Popover's Observer to fire and reposition the anchor element.
I tried thinking of a deterministic way, but it'd be specific to the Popover story, and this spec goes over several other components too.
I couldn't query for an `.ods-popover-anchor` because is not every story that would have one.

Any ideas?

Reply:
Did some more testing, turns out the theme class is only applied to the iframe's body after one React cycle so I could remove the `cy.wait`.
If it becomes flaky later, we can think of an additional check to perform.